website/settings/defaults.py

```diff
@@ -578,6 +585,10 @@ class CeleryConfig:
                 'task': 'scripts.generate_sitemap',
                 'schedule': crontab(minute=0, hour=5),  # Daily 12:00 a.m.
             },
+            'check_crossref_doi': {
+                'task': 'osf.management.commands.check_crossref_dois',
```

Comment:
```suggestion
                'task': 'management.commands.check_crossref_dois',
```

Reply:
I believe it's using the name you provided to the task decorator and not the actual path to the script, so I think this will work better.